test.py

```diff
@@ -0,0 +1,8 @@
+from pygraphviz import *
```

Comment:
Is this file just a test of pygraphviz functionality? If so, we don't need this in the master branch - you should remove it.